File: app/src/main/java/com/example/myapplication/Receipts.java
package com.example.myapplication;

import android.content.Intent;
import android.os.Bundle;
import android.os.CountDownTimer;
import android.view.View;
import android.widget.ImageButton;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class Receipts extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.receipts);
        ImageButton back = (ImageButton)findViewById(R.id.dann2);

        back.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                back.setImageResource(R.drawable.knopkaser);
                new CountDownTimer(300, 1000) {
                    public void onTick(long millisUntilFinished) {
                    }

                    public void onFinish() {
                        back.setImageResource(R.drawable.knopkaser_nazh);
                        Intent i = new Intent(Receipts.this, Menu.class);
                        startActivity(i);
                    }
                }.start();
            }
        });

        //Бэтмен аккордеон на ютубе бомба
    }
}

File: app/src/main/java/com/example/sipirecipes/Receipts.kt
package com.example.sipirecipes

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.sipirecipes.R
import android.widget.ImageButton
import android.os.CountDownTimer
import android.content.Intent
import android.view.View

class Receipts : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.receipts)
        val back = findViewById<View>(R.id.dann2) as ImageButton
        back.setOnClickListener {
            back.setImageResource(R.drawable.knopkaser)
            object : CountDownTimer(300, 1000) {
                override fun onTick(millisUntilFinished: Long) {}
                override fun onFinish() {
                    back.setImageResource(R.drawable.knopkaser_nazh)
                    val i = Intent(this@Receipts, Menu::class.java)
                    startActivity(i)
                }
            }.start()
        }

        //Бэтмен аккордеон на ютубе бомба
    }
}